python/lsst/ip/isr/assembleCcdTask.py

```diff
@@ -231,6 +232,37 @@ def getNextExposure(amp):
         for amp in ccd:
             inMI = getNextExposure(amp).getMaskedImage()
             assemble(outMI, inMI, amp)
+
+        if not self.config.doTrim:      # we need to update the Detector
```

Comment:
A few more words in the comment would be much appreciated (and moving it to the next line). This is the sort of thing I have in mind (though some of this may already be in the help for `config.doTrim` -- I haven't found that yet):
`Update the amp info table in the Detector to describe untrimmed amplifier data assembled into one image, with the same flips as trimmed assembled data.`